cmd/wallet-web/src/pages/Vaults.vue

```diff
@@ -24,7 +24,7 @@
         :vault-id="vault.id"
       >
         <slot v-if="vault.name == 'Default Vault'">
-          <flyout-menu type="vault">
+          <flyout-menu type="vault" id="defaultVaultFlyoutMenu">
             <flyout-menu-list>
               <flyout-menu-button
                 id="renameVault"
```

Comment:
Ohhh I just noticed we still have these hard-coded `id`s for `flyout-menu-button`s, we should not do that either

Reply:
As we discussed these flyout menu are rendered one at a time , so these ids should not be a problem